### ISSUE TITLE
[New rule] Capitalize comments

### ISSUE THREAD

**Issue by NickBalnaves** [dart-code-checker/dart-code-metrics#566]
A rule to ensure comments are formatted as sentences https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences. 

There is an existing eslint rule which capitalizes comments: https://eslint.org/docs/rules/capitalized-comments. 

Good
```dart
// Not if there is nothing before it.
if (_chunks.isEmpty) return false;
```

Bad
```dart
// not if there is nothing before it.
if (_chunks.isEmpty) return false;
```

**Comment by dkrutskikh** [comment]
Available in 4.11.0 release!
